docs/release-notes/8.13.asciidoc

```diff
@@ -1,6 +1,15 @@
 [[release-notes-header-8.13.0]]
 == 8.13
 
+[discrete]
+[[release-notes-8.13.2]]
+=== 8.13.2
+
+[discrete]
+[[bug-fixes-8.13.2]]
+==== Bug fixes
+* Fixes a bug that automatically checked checkboxes on the Alerts page when you clicked filter labels ({kibana-pull}179610[#179610]).
```

Comment:
Adding here since unable to push from my local remote atm.

```suggestion
* Fixes alert dashboard filters  ({kibana-pull}179911[#179911]).  
* Fixes a bug that automatically checked checkboxes on the Alerts page when you clicked filter labels ({kibana-pull}179610[#179610]).
```